src/redux/actions/userActions.js

```diff
@@ -138,14 +135,11 @@ export const loginUserAsync = (
 
     try {
       // authenticate user
-      const authUserResp = await axios.post(
-        "http://localhost:3001/api/users/login",
-        {
-          email,
-          password,
-          isGoogleLogin: user.data.isGoogleUser,
-        }
-      );
+      const authUserResp = await axios.post(`${BASE_URL}/login`, {
```

Comment:
im assuming the base url is like http://localhost:3000/api so here you're missing the /users part of the path